packages/ssx-sdk/src/core.ts

```diff
@@ -18,8 +18,19 @@ import {
 export class SSXInit {
   /** Extensions for the SSXClientSession. */
   private extensions: SSXExtension[] = [];
+  private executionEnvironment: 'node' | 'browser';
 
-  constructor(private config?: SSXClientConfig) { }
+  constructor(private config?: SSXClientConfig) { 
+    this.executionEnvironment = typeof process.versions.node !== 'undefined' ? 'node' : 'browser';
```

Comment:
Make this configurable perhaps? I would like to use it with deno or in other context where we do not have a browser wallet.